surveys/sphinx-survey-2025.yml

```diff
@@ -313,12 +305,10 @@ questions:
   - "Whatever comes with Sphinx by default"
   - "Alabaster"
   - "Furo"
-  - "Material design"
+  - "Material for Sphinx"
```

Comment:
This could be either of [material](https://github.com/bashtage/sphinx-material) or [immaterial](https://jbms.github.io/sphinx-immaterial/), but only the latter is maintained.